index.js

```diff
@@ -15,6 +15,7 @@ import App from 'src/app/App'
 import * as Sentry from '@sentry/react-native'
 import 'react-native-gesture-handler'
 import { Text, TextInput } from 'react-native'
+import '@walletconnect/react-native-compat'
```

Comment:
This is how several other wallets import '@walletconnect/react-native-compat`: [search](https://github.com/search?q=%40walletconnect%2Freact-native-compat+%22index.js%22&type=code). 

[WalletConnect RN CLI Wallet](https://github.com/WalletConnect/react-native-examples/tree/main/wallets/rn_cli_wallet)
```JavaScript
// index.js
import './polyfills';

// polyfills.ts
import '@walletconnect/react-native-compat';
```

```

Reply:
is this directly related to the fix?

Reply:
When the import was in the `src/walletconnect/saga`, I was seeing more issues with pairings.